enhancements/olm/operator-bundle.md

```diff
@@ -105,17 +105,30 @@ An operator author can also specify the version of the format used inside the bu
 operators.operatorframework.io.bundle.mediatype: "registry+v1"
 ```
 
+###### Graph Metadata
+An additional point of concern is the fact that the current [operator-registry manifest format](https://github.com/operator-framework/operator-registry#manifest-format) also has a multi-version aggregation file `package.yaml` that describes data about the channels that individual versions are subscribed to. Given that we are splitting up these manifests into separate objects, we now need a way to define how an individual version subscribes to a given channel. To do this, we will create a new metadata file that defines the channels explicitly for each version called `channels.yaml`
+
+*channels.yaml*
+```yaml
```

Comment:
Why not have these be labels / annotations.yaml entries?

```
operators.operatorframework.io.bundle.package=etcd
operators.operatorframework.io.bundle.channels=alpha,stable
```
It doesn't help much on-cluster because we have to go through crio, but on the registry side it saves us some blob unpacking.

Reply:
I actually like this idea given it will simplify things even further for bundle.

Reply:
But the registry will still need to unpack all of the other files, so I'm not sure it really saves us anything. @dinhxuanvu @ecordell Could you elaborate on why this will simplify it? Just because don't have to worry about multiple files?

Reply:
From my perspective, we're adding a stripped-down version of `package.yaml` that only contains package name and some channels. Those can be easily added to `annotations.yaml` as the `annotations.yaml` itself is a new metadata file that we require as a part of this bundle image enhancement (similar to what you are doing with `channels.yaml` in this PR). Two new things can be combined into one. As a result, we only require 1 single metadata file. That sounds beneficial.
To be honest, Evan's approach requires some medium changes from the existing PR that I already have compared to your approach, Kevin, which requires only minor adjustment. I'm totally fine to make changes on either way as long as we can agree on one or the other.

Reply:
We discussed offline yesterday, but the idea is that, in the future, we may have access to  labels directly on a cluster and can deprecate the in-image annotations.yaml.